workers/unity/Packages/com.improbable.gdk.querybasedinteresthelper/InterestHelper.cs

```diff
@@ -0,0 +1,220 @@
+using System.Collections.Generic;
+using System.Linq;
+using Improbable.Gdk.Core;
+
+namespace Improbable.Gdk.QueryBasedInterest
+{
+    /// <summary>
+    ///     Utility class to help construct Interest component snapshots.
+    /// </summary>
+    public class InterestHelper
+    {
+        private readonly Dictionary<uint, ComponentInterest> interest;
+
+        private InterestHelper(Dictionary<uint, ComponentInterest> interest)
```

Comment:
default value `null`, otherwise I wouldn't know that its a valid value to use. Saves passing it in in `Begin`